dts/arm/nordic/nrf52840.dtsi

```diff
@@ -60,6 +60,16 @@
 		qdec-0 = &qdec;
 	};
 
+	sw_pwm: sw-pwm {
+		compatible = "nordic,nrf-sw-pwm";
+		timer-instance = <2>;
+		clock-prescaler = <0>;
+		ppi-base = <14>;
+		gpiote-base = <0>;
+		status = "disabled";
+		label = "SW_PWM";
+	};
+
```

Comment:
Is it possible/desirable to have a nrf.dtsi that declares sw_pwm: sw-pwm {} with default values and then have individual SoC's override the values that they need to set differently?

Reply:
Yes, that is possible. 
Steps: 
- add new file nrf.dtsi in same folder as nrf52832.dtsi. 
- Define sw_pwm: ... node in this file instead of nrf52832.dtsi
-- Leave out timer-instance, clock-prescaler, ppi-base, and gpiote-base.
- In nrf52832.dtsi, include modify this node by writing &sw_pwm { ... and then set timer-instance, clock-prescaler, ppi-base, and gpiote-base here.
- Works the same from application - no changes there. 

If this is desirable, i don't know?

Reply:
Yes, I think it's a good idea to have a common definition of the node and override only the parts that need be changed for particular SoCs.

Reply:
Done. In the common nrf5.dtsi, i set timer-instance, clock-prescaler, ppi-base, gpiote-base all to 0. If we want to keep this as the default config for nrf52 instead that can be done, but i find it "cleaner" if both nrf51 and nrf52 actually change these values to what they should be